operator/.gitignore

```diff
@@ -0,0 +1,25 @@
+
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+bin
+testbin/*
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Kubernetes Generated files - skip generated files, except for vendored files
+
+!vendor/**/zz_generated.*
```

Comment:
? I don't understand, this line should imply we ignore generated files, but they are still part of some following commits :thinking:

Reply:
This gitignore come from operator-sdk.

To me it's the opposite `!`, we do not ignore the `zz_generated.*` files.

I think it's either a leftover in operator-sdk because they usually git ignore the `vendor` directory, either just here to ensure we do not git ignore those specific files

Reply:
Right, that's strange, the comment is quite misleading.